index/server/entrypoint.sh

```diff
@@ -10,7 +11,27 @@ if [ ! -e "$DEVFILE_INDEX" ]; then
     exit 1
 fi
 
+# Start the registry viewer
+npm start &
+
+# Wait for server to start
+until $(curl --output /dev/null --silent --head --fail http://localhost:3000/viewer); do
+    printf '.'
+    sleep 1
+done
+
+# Copy over the registry viewer config
+# cp -rf /viewer-config/devfile-registry-hosts.json /app/config/
+
 # Start the index server
 /registry/index-server
 
-fg %1
+
+# Try to force the registry viewer pages to generate
+# for n in {1..10}; do
+# 	echo "Sleeping $n"
+# 	sleep 1
+# 	curl http://localhost:8080/viewer || true
+# done
+
+tail -f /dev/null
```

Comment:
just a thought, could the index server and the viewer pipe their output to a file and we can tail that file for logs?

Reply:
So we don’t need to pipe the output since the output of the server is preserved with the`&`, so the output is still visible in the container logs.

But, this line is actually redundant, since we’re not running the index server in the background, so it can be removed.

Reply:
Fixed.